HomeUI/src/views/apps/Management.vue

```diff
@@ -4991,6 +4991,9 @@ export default {
           // time to encrypt
           // eslint-disable-next-line no-restricted-syntax
           for (const component of this.appUpdateSpecification.compose) {
+            component.environmentParameters = component.environmentParameters.replace('\\“', '\\"');
```

Comment:
Can you explain what it is for?

Reply:
We have lots of support tickets, where apps are specified on the network but not running in any node.
The reason it's because they have copied the specifications from medium or guides, where the char for a string is not valid, the UI allows it, but when the app is selected to install on a node fails saying the JSON is not valid. And because if we gave an error most probably people would not understand the reason, I think this should be just fine.